src/main/java/smarthome/models/User.java

```diff
@@ -0,0 +1,90 @@
+package smarthome.models;
+
+/**
+ * Class of type User.
+ */
+public class User{
+    /**
+     * Creating a private int called id.
+     */
+    private int id;
+    /**
+     * Creating a private String called name.
+     */
+    private String name;
+    /**
+     * Creating a private enum called role with constants Parent, Child, Guest and Stranger.
+     */
+    private enum role {Parent, Child, Guest, Stranger};
```

Comment:
This enum isn't set properly.

Also, despite the small optimization we get from using an enum, the optimization we get is negligible, storing the roles as Strings is simpler, and it'll avoid having to convert the value from enum to String when communicating to the dashboard. Most web frameworks store roles as Strings. There's nothing wrong with it.

- [ ] Replace enum with a String variable. We'll validate the value before setting it in "SimulationParameters".

Reply:
Replaced enum with String Variable